packages/store-sdk/src/session/Provider.tsx

```diff
@@ -0,0 +1,76 @@
+import React, { createContext, useMemo } from 'react'
+import type { FC } from 'react'
+
+import { useLocalStorage } from '../storage/useLocalStorage'
+
+export interface Currency {
+  code: string // Ex: USD
+  symbol: string // Ex: $
+}
+
+export interface User {
+  id: string // user id
+}
+
+interface Session {
+  locale: string // en-US
+  currency: Currency
+  country: string // BRA
+  channel: string | null
+  region: string | null
+  priceTable: string | null
```

Comment:
Are these industry terms or do we have any VTEX bias on their naming?

Reply:
I'm not sure about region and priceTable, however I'm sure about channel. I'll leave only the channel one